src/app/icons/communication.tsx

```diff
@@ -0,0 +1,8 @@
+const Communication = () => (
```

Comment:
Instead of using the svg can you guys use one of the "inbox" icons from iconify? Just to keep it consistent!